tests/slow/src/test/scala/tests/scalacli/ScalaCliSuite.scala

```diff
@@ -189,30 +189,52 @@ class ScalaCliSuite extends BaseScalaCliSuite(V.scala3) {
       )
 
       _ <- server.didOpen("MyTests.sc")
-      _ = assertNoDiff(
-        client.syntheticDecorations,
-        s"""#!/usr/bin/env -S scala-cli shebang --java-opt -Xms256m --java-opt -XX:MaxRAMPercentage=80 
-           |//> using scala "$scalaVersion"
-           |//> using lib "com.lihaoyi::utest::0.7.10"
-           |//> using lib com.lihaoyi::pprint::0.6.6
-           |
-           |import foo.Foo
-           |import utest._
-           | 
-           |pprint.log[Int](2)(generate, generate) // top-level statement should be fine in a script
-           |
-           |object MyTests extends TestSuite {
-           |  pprint.log[Int](2)(generate, generate)
-           |  val tests: Tests = Tests {
-           |    test("foo") {
-           |      assert(2 + 2 == 4)
-           |    }
-           |    test("nope") {
-           |      assert(2 + 2 == (new Foo).value)
-           |    }
-           |  }
-           |}
-           |""".stripMargin,
+      _ <- server.assertInlayHints(
+        "MyTests.sc",
+        s"""|#!/usr/bin/env -S scala-cli shebang --java-opt -Xms256m --java-opt -XX:MaxRAMPercentage=80 
```

Comment:
Do you need to pass the whole file input, won't that be available in `buffers` in testing server?

Reply:
I think most tested methods do it this way. 

I've changed the signature so the tests should be more readable.